### ISSUE TITLE
Support Red Hat Enterprise and related Linux systems

### ISSUE THREAD

**Issue by yllierop** [Chia-Network/chia-blockchain#41]
Fix [/lib/chiavdf/fast_vdf](https://github.com/Chia-Network/chia-blockchain/blob/master/lib/chiavdf/fast_vdf/install_child.sh)

```
echo "enable_all_instructions: $enable_all_instructions"
enable_all_instructions: 1

if [[ "$OSTYPE" == "linux-gnu" ]]; then
    sudo apt-get install libgmp3-dev -y
    sudo apt-get install libflint-dev -y
    sudo apt install libboost-all-dev
    opt_flags="-O3"
    link_flags="-no-pie -lgmpxx -lgmp -lflint -lpthread -lboost_system"
    compile_flags="-std=c++1z -D VDF_MODE=0 -D ENABLE_ALL_INSTRUCTIONS=$enable_all_instructions -no-pie -march=native"
elif [[ "$OSTYPE" == "darwin"* ]]; then
    opt_flags=""  # Catalina clang issues
    link_flags="-no-pie -lgmpxx -lgmp -lflint"
    compile_flags="-std=c++1z -D CHIAOSX=1 -D VDF_MODE=0 -D ENABLE_ALL_INSTRUCTIONS=$enable_all_instructions -no-pie -march=native"
else
    opt_flags="-O3"
    link_flags="-no-pie -lgmpxx -lgmp -lflint"
    compile_flags="-std=c++1z -D VDF_MODE=0 -D ENABLE_ALL_INSTRUCTIONS=$enable_all_instructions -no-pie -march=native"
fi
sudo: apt-get: command not found
sudo: apt-get: command not found
sudo: apt: command not found
```

**Comment by hoffmang9** [comment]
We haven't ported to yum/rpm based systems but if you know much about yum it's pretty easy. We will have this likely in the next release and I'll see if I can get you the instructions beforehand

**Comment by hoffmang9** [comment]
Support officially available - likely in the alpha-1.3 release
https://github.com/Chia-Network/chia-blockchain/tree/windows
